models/cam-se/model_mod.f90

```diff
@@ -1747,7 +1780,8 @@ subroutine build_dry_mass_pressure_columns(ens_handle, ens_size, nlevels, column
       half_pressure(k + 1) = half_pressure(k) + &
          !gravity * (a_width(k)*dry_mass_top + b_width(k)*dry_mass_sfc(n)) * sum_dry_mix_ratio(k, n)
          ! SENote: NEXT LINE IS BELIEVED TO BE CORRECT BUT NEEDS TO BE VETTED WITH CGD
-         gravity * (a_width(k)*dry_mass_top / grid_data%hyai%vals(1) + b_width(k)*dry_mass_sfc(n)) * sum_dry_mix_ratio(k, n)
+         gravity * (a_width(k)*dry_mass_top/grid_data%hyai%vals(1) + b_width(k)*dry_mass_sfc(n)) &
+                 * sum_dry_mix_ratio(k, n)
```

Comment:
Is this a change to the calculation? Or are you just wrapping the line.
If it is just a line wrap change, I would not do this change in this pull request.